### DISCUSSION TITLE
Syntax Highlighting Support for Typst in GitHub

### DISCUSSION THREAD

**Discussion by chengluyu** [typst/typst#2118] [General]
I noticed that GitHub currently does not support syntax highlighting for Typst. Viewing and understanding Typst code on GitHub is a little bit challenging. GitHub uses a library called Linguist to handle syntax highlighting. Linguist supports many languages, and it has a process for adding new languages, which is available [here](https://github.com/github-linguist/linguist#adding-a-language). Do you plan to provide syntax highlighting support for Typst files on GitHub?

**Comment by frozolotl** [comment]
It was [merged yesterday](https://github.com/github-linguist/linguist/pull/6379).
